### PR TITLE
Added the event on the end turn button

### LINKED ISSUE #22: Handle turns
Implement finishing the player's turn and the tail of the game loop.

### PR DESCRIPTION
**Pull request by KartikPat250905**

### CODE REVIEW

File: src/frontend/src/components/turnutils.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { matchEndScene } from "./winandloss";
+
+export function Myloop(GameState)
+{
+	if (isEndGame(GameState)){
+		//Add the scene switch here.
+		return true;
+	}
+	MoveDracula(GameState);//This is stupid path decider add wangs funcion here.
```
Comment (ktnlvr) [inline src/frontend/src/components/turnutils.js]: Function name capitalized

File: src/frontend/src/components/turnutils.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import { matchEndScene } from "./winandloss";
+
+export function Myloop(GameState)
```
Comment (ktnlvr) [inline src/frontend/src/components/turnutils.js]: Function name and argument capitalized

File: src/frontend/src/main.js [hunk 3]
```diff
@@ -58,13 +59,15 @@ async function setupGame() {
...
   ]);
-
   render(cameraControls, spotlightHelper);
   /* Example actions
   logInfo("Hello this is a fucking cat.");
   matchEndScene("loss");
   characterDeath(document.querySelector('[char-id="2"]'));
   */
+  document.querySelector(".end-turn-btn").addEventListener('click',() => {
+    Myloop(GameState);
```
Comment (ktnlvr) [inline src/frontend/src/main.js]: Why is a function called loop called when clicking?

File: src/frontend/src/features.css [hunk 4]
```diff
@@ -127,3 +127,41 @@
...
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+  position: absolute;
+  top: 50%;
+  left: 68%;
+  opacity: 0.85;
+}
+
+.end-turn-btn {
```
Comment (ktnlvr) [inline src/frontend/src/features.css]: Maybe use `button` instead

File: src/frontend/src/components/turnutils.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+		matchEndScene("loss");
+	if (isEndGame(GameState)){
+		//switch scene here again.
+		return true;
+	}
+}
+
+// Checks if the game has reached its end condition.
+// Returns true if any player shares the same airport as Dracula.
+export function isEndGame(GameState) {
```
Comment (ktnlvr) [inline src/frontend/src/components/turnutils.js]: Capital variable name

File: src/frontend/src/main.js (hunk 3 above)
Comment (KartikPat250905) [reply src/frontend/src/main.js]: this is just the convinient function, it calls all the other function in the sequence from the flowchart it is just for testing and stuff.

**Pull request merged**